libs/utils/decorate.js

```diff
@@ -42,12 +42,25 @@ export function decorateIconStack(el) {
   });
 }
 
-export function decorateIconArea(el) {
+export function decorateIconArea(el, headings = null) {
   const icons = el.querySelectorAll('.icon');
   icons.forEach((icon) => {
     icon.parentElement.classList.add('icon-area');
     if (icon.textContent.includes('persona')) icon.parentElement.classList.add('persona-area');
   });
+  // better icon area
+  if (headings) {
+    headings.forEach((h) => {
+      const hPrevElem = h.previousElementSibling;
+      if (hPrevElem?.childElementCount) {
+        let picCount = 0;
+        [...hPrevElem.children].forEach((child) => {
+          if (child.nodeName === 'PICTURE') picCount += 1;
+        });
```

Comment:
candidate for reduce
```suggestion
        let picCount = [...hPrevElem.children].reduce((rdx, child) => {
          if (child.nodeName === 'PICTURE') rdx += 1;
          return rdx;
        }, 0);
```